### DISCUSSION TITLE
Correcting the Commit Description/Message

### DISCUSSION THREAD

**Discussion by CodegoErgoSum** [community/community#36033] [Other Feature Feedback, Questions, & Ideas]
Hi, Everybody!

I am new to GitHub.  When committing my changes, I made a minor mistake in the description/message field.  How can I correct the commit description/message?

Please advise.

Thank-you!

**Comment by airtower-luna** [comment]
Within the GitHub web interface the answer is: You can't.

If you have a local copy of the repository you can [rewrite the repository history](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History) to change it (easy for the latest commit, harder for older ones). The catch with rewriting history is that it can cause trouble *if* you've already shared it, anyone who has a copy of the repository would have to manually accept the new history. It's common to rewrite history on a feature branch before merging, after tends to be frowned upon unless there is a very strong reason.

**Comment by CodegoErgoSum** [comment]
This issue has been resolved.
